### PR TITLE
Added a new SQL question

### PR DESCRIPTION
**Pull request by btrezzy**
**Checklist:**
- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org/).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (not a vague title like Update index.md)

### CODE REVIEW

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -752,6 +752,16 @@ const sqlQuiz = [
     Explanation:
       "CHAR holds a fixed length string (up to 255 characters), while VARCHAR and TEXT hold variable length strings.",
     Link: "https://www.freecodecamp.org/news/sql-data-types-mysql/"
+  },
+    {
+    Question: "In SQL, which of the following LIKE operator finds any values that have 'or' at any position?",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:757]:
```suggestion
    Question: "In SQL, which of the following wildcards combined with the LIKE operator finds any value that has 'or' at any position?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by btrezzy** [comment]
Changes made, thanks for the feedback!

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**